controllers/operatorconfig_controller.go

```diff
@@ -41,9 +41,11 @@ import (
 )
 
 const (
-	Finalizer                = "openshift.konveyor.crane"
-	OwnerConfigName          = "openshift-migration-toolkit"
-	InvalidNameConditionType = "InvalidName"
+	Finalizer                           = "openshift.konveyor.crane"
+	OwnerConfigName                     = "openshift-migration-toolkit"
+	InvalidNameConditionType            = "InvalidName"
+	ReconciledConditionType             = "Reconciled"
+	ErrorCreatingResourcesConditionType = "ErrorCreatingResources"
```

Comment:
I'm thinking that what we may want to do is to have a single "basic" type that is something like `ReconcileCompleted` and all of these would be "reason"s. For example:

* Invalid name -> Condition `ReconcileCompleted` is `False` with reason `InvalidName`
* Error creating resource -> Condition `ReconcileCompleted` is `False` with reason `ErrorCreatingResources`
* Complete reconcile -> Condition `ReconcileCompleted` is `True` with reason `ReconcileComplete`
additionally we could add messages to make them more useful.

This would give us a thoroughly useful, albeit basic, condition on the CR for users to get a sense of if it's ready or not.